website/docs/r/firewall.html.md

```diff
@@ -70,9 +70,9 @@ The following arguments are supported in the inbound and outbound rule blocks:
 
 * `protocol` - (Required) The network protocol this rule controls.
 
-* `ipv4` - (Optional) A list of IP addresses, CIDR blocks, or 0.0.0.0/0 (to allow all) this rule applies to.
+* `ipv4` - (Optional) A list of IPv4 addresses or networks. Must be in IP/mask format.
 
-* `ipv6` - (Optional) A list of IPv6 addresses or networks this rule applies to.
+* `ipv6` - (Optional) A list of IPv6 addresses or networks. Must be in IP/mask format.
 
```

Comment:
`label`, `action` docs missing here

`inbound_policy`, `outbound_policy` are needed in Argument Reference as well. (can't add review comment there because no changes)